frontend/components/httpsUrlInput.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function onInvalidUrl(event) {
+  event.target.setCustomValidity('Please enter a URL that starts with "https://"');
+}
+
+function onUrlInput(event) {
+  event.target.setCustomValidity('');
```

Comment:
This is needed to "unset" the custom validity message. Kind of weird, but that's how the validity API works 🤷‍♀️